src/04_printing.py

```diff
@@ -11,7 +11,17 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
+print("%d %d %s" % (x, y, z))
```

Comment:
Here it is asking to return `y is 2.25`, so you need to round, you do this by using ` %.2f`